### PR TITLE
Add a WatchAllPods function

### PR DESCRIPTION
**Pull request by KoishiKomeiji**

### CODE REVIEW

**Comment by mpuncel** [comment]
nit: our graphQL stuff is internal, don't mention it in public commit messages

File: pkg/kp/kv.go [hunk 1]
```diff
@@ -378,6 +379,39 @@ func (c consulStore) WatchPods(
...
+func (c consulStore) WatchAllPods(
+	podPrefix PodPrefix,
+	quitChan <-chan struct{},
+	errChan chan<- error,
+	podChan chan<- []ManifestResult,
+) {
+	defer close(podChan)
+
+	kvPairsChan := make(chan api.KVPairs)
+	go consulutil.WatchPrefix(string(podPrefix), c.client.KV(), kvPairsChan, quitChan, errChan, 0)
```
Comment (mpuncel) [inline pkg/kp/kv.go]: you should expose the `pause` (i.e. `0`) parameter to `WatchPrefix` to the caller, our /intent tree is over 40MB which means the fetch might take on the order of 10s of seconds. We can tune consul load by pausing between updates as necessary

**Comment by mpuncel** [comment]
looks good, once you expose the pause parameter

File: pkg/kp/kv.go (hunk 1 above)
Comment (KoishiKomeiji) [inline pkg/kp/kv.go]: Alright I changed it to 10 seconds.

**Comment by KoishiKomeiji** [comment]
It's been changed to take in a parameter now.

**Comment by mpuncel** [comment]
👍

**Comment by KoishiKomeiji** [comment]
Thanks!

**Pull request merged**